tools/dnt_preset_editor.html

```diff
@@ -91,9 +91,62 @@
 td.routingSlot {
 	text-align: center;
 }
+.lua-controls {
+    margin: 10px 0;
+}
+.lua-editor {
+    margin: 10px 0;
+	display: none;
+}
+.lua-editor-header {
+	display: flex;
+	gap: 10px;
+	align-items: center;
+	margin: 20px 0;
+}
+#luaScript {
+    width: 100%;
+	max-width: 800px;
+	min-height: 1024px;
+    font-family: monospace;
+    background-color: #f8f8f8;
+    border: 1px solid #ddd;
+    padding: 10px;
+}
+.map_ui_lua {
+    display: none;
+}
+
+/* New styling for the slot grid */
+#slotGrid {
+    margin: 10px 0;
+}
+.slotRow {
+    margin-bottom: 5px;
+}
+.slotBtn {
```

Comment:
The slot buttons seem overly large to me, and there's a lot of wasted space for "Slot:" on a line by itself.

At the risk of feature creep, people are immediately going to ask for the buttons to be shaded or something depending on whether there's an algorithm in the slot or not.

Reply:
Should I revert it or try improving it? Could people say the same for the dropdown as it has 32 slots?

Reply:
I would suggest
- make the buttons a bit smaller and get rid of the mostly empty line.
- only show buttons for filled slots. Pressing a button (or selecting the drop-down) for a slot that's not there serves no purpose.

Reply:
Sounds good, thank you